### ISSUE TITLE
Thank you very much

### ISSUE THREAD

**Issue by 02monkey** [kakopappa/esp8266_smartconfig_flutter_example#1]
Thank you very much, I novice, want to go to the development of the Internet of Things, first see if Flutter supports the Internet of Things, unexpectedly found you, thank you again!

**Comment by kakopappa** [comment]
Yes it does.  Whatever that does not support can be implemented using plugins.  I am working on few IOT projects using flutter. can make beautiful apps with Flutter
